app/views/notifier/contacts_birthday.markerb

```diff
@@ -0,0 +1,6 @@
+
+<%= @notification.person.name %> <%= t('.birthday') %>
+
+[<%= t('.view_profile', :name => @notification.person.name) %>][1]
```

Comment:
I just saw that pronto doesn't check this file, but can you use double quotes and ruby 1.9 hash syntax here too? (in the whole file)